src/main/java/com/ibm/crypto/plus/provider/ock/Digest.java

```diff
@@ -92,10 +88,8 @@ void getContext() throws OCKException {
             }
         }
 
-        if (this.nativeResources != null) {
+        if (this.digestId != 0) {
```

Comment:
Not sure why a class was used here. Talked with Karthik and simple field is much better, since the class causes memory to be held on too and can cause an out of memory issue.